components/brave_ads/core/internal/serving/eligible_ads/exclusion_rules/exclusion_rules_base.h

```diff
@@ -42,7 +42,7 @@ class ExclusionRulesBase {
   std::vector<std::unique_ptr<ExclusionRuleInterface<CreativeAdInfo>>>
       exclusion_rules_;
 
-  std::set<std::string> uuids_;
+  std::unordered_set<std::string> uuids_;
```

Comment:
is it better to use std::set here?

Reply:
For real-world data we found that flat_set was approx. 8% faster. However, it has been decided to followup across ads to investigate and optimize where needed.